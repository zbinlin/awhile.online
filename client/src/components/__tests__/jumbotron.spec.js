"use strict";

/* eslint-env jest */
import { h } from "preact";
import renderer from "react-test-renderer";
import Jumbotron from "../jumbotron.jsx";

it("Jumbotron renders correctly", () => {
    const tree = renderer.create(
        <Jumbotron />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
