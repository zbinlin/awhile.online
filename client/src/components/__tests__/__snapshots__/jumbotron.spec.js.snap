exports[`test Jumbotron renders correctly 1`] = `
<div
  className="jumbotron">
  <div
    className="mid">
    <div
      className="logo">
      <span
        className="name">
        Awhile
      </span>
      <span
        className="dot">
        .
      </span>
      <span
        className="domain">
        Online
      </span>
    </div>
  </div>
</div>
`;
